### PR TITLE
MNT: monitor fixes

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

File: bluesky/run_engine.py [hunk 1]
```diff
@@ -1303,7 +1303,9 @@ def _monitor(self, msg):
         if msg.args:
             raise ValueError("The 'monitor' Msg does not accept positional "
                              "arguments.")
-        name = msg.kwargs.get('name')
+        kwargs = dict(msg.kwargs)
+        name = kwargs.pop('name', new_uid()[:6] + '_monitor')
```
Comment (klauer) [inline bluesky/run_engine.py]: I thought we had like a `new_short_uid` somewhere...

Comment (tacaswell) [inline bluesky/run_engine.py]: we probably do...

File: bluesky/plans.py [hunk 2]
```diff
@@ -1164,7 +1160,8 @@ def monitor_during_wrapper(plan, signals):
     --------
     `bluesky.plans.fly_during_wrapper`
     """
```
Comment (danielballan) [inline bluesky/plans.py]: Agree that my scheme for defaulting name was not configurable enough. Could we keep `sig.name + '-monitor` as a default?

File: bluesky/run_engine.py [hunk 3]
```diff
@@ -1248,7 +1248,9 @@ def _monitor(self, msg):
         if msg.args:
             raise ValueError("The 'monitor' Msg does not accept positional "
                              "arguments.")
-        name = msg.kwargs.get('name')
+        kwargs = dict(msg.kwargs)
+        name = kwargs.pop('name', short_uid('monitor'))
```
Comment (danielballan) [inline bluesky/run_engine.py]: Good catch.

File: bluesky/plans.py (hunk 2 above)
Comment (danielballan) [inline bluesky/plans.py]: I read the diff backward. Carry on!

**Comment by danielballan** [comment]
👍

**Pull request merged**